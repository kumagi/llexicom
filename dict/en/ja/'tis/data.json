{"word":"'tis","priority":"★☆☆","meanings":[{"part_of_speech":"contraction","definition":"it is (古風な表現)","english_definition":"Archaic contraction of 'it is'.","examples":[{"sentence":"'Tis a far, far better thing that I do, than I have ever done.","translation":"私が今から行うことは、これまで行ったどんなことよりも、はるかに、はるかに素晴らしいことなのだ。","note":"チャールズ・ディケンズ『二都物語』の有名な一節。"}],"collocations":["'tis the season (よくクリスマスシーズンに使われる)"],"synonyms":["it is"],"antonyms":[]}],"etymology":{"value":"'Tis は 'it is' の短縮形であり、アポストロフィは脱落した文字を示します。この短縮形は、16世紀頃から英語で使用されてきました。","priority":"★★☆"},"pronunciation":{"ipa":"/tɪz/","syllables":"tis"},"inflection":{"notes":"短縮形のため、活用はありません。"},"usage_notes":{"explanation":"'Tis は 'it is' の古風な短縮形であり、現代英語ではほとんど使用されません。文学作品や、クリスマスキャロルなどの伝統的な文脈で見られることがあります。日常会話で使用すると、古めかしい印象を与えます。","priority":"★★★"},"common_mistakes":{"examples":[{"incorrect":"✗ Tis are a good day.","correct":"✓ 'Tis a good day. / ✓ It is a good day.","note":"'Tis は 'it is' の短縮形なので、主語と動詞を別に置くことはできません。"}],"priority":"★★☆"},"related_words":{"derivatives":[],"idioms_phrases":["'Tis the season to be jolly (陽気になる季節だ - クリスマスキャロルの一節)"],"priority":"★★☆"},"level_frequency":{"CEFR":"N/A (現代英語ではほとんど使用されない)","frequency_google_ngram":"非常に低い頻度 (Very low frequency) - 古風な表現。","priority":"★☆☆"},"readability_explanation":{"level":"C2","text":"「**'tis**」は「it is」の古い言い方で、現代の英語ではほとんど使われません。昔の本や詩、またはクリスマスの歌などに見かけることがあります。普段の会話で使うと、少し古くさい感じがします。"},"example_sentences":[{"sentence":"'Tis a beautiful day for a walk.","translation":"散歩にはもってこいの美しい日だ。","type":"contraction","meaning_category":"it is (古風)"}]}