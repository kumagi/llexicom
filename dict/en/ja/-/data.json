{"word":"-","priority":"★★★","meanings":[{"part_of_speech":"punctuation","definition":"ハイフン、ダッシュ","english_definition":"A punctuation mark used to join words or parts of words, to separate items in a list, or to indicate a pause or break in speech.","examples":[{"sentence":"Use a hyphen to join two words to form a compound adjective, such as 'well-known'.","translation":"「よく知られた」のように、ハイフンを使って2つの単語を結合して複合形容詞を形成します。"},{"sentence":"The dash can be used to indicate an abrupt break in thought.","translation":"ダッシュは、思考の突然の中断を示すために使用できます。"},{"sentence":"My favorite colors are blue, red, and green.","translation":"私のお気に入りの色は、青、赤、緑です。"}],"collocations":["hyphenated word (ハイフンでつながれた単語)","em dash (全角ダッシュ)","en dash (半角ダッシュ)","minus sign (マイナス記号)"],"synonyms":["hyphen","dash","minus"],"antonyms":[]}],"etymology":{"value":"The hyphen derives from the Greek 'hyphen', meaning 'together'. The dash evolved later as a longer form of the hyphen.","priority":"★★☆"},"pronunciation":{"ipa":"N/A","syllables":"N/A"},"inflection":{"notes":"Not applicable as it is a punctuation mark."},"usage_notes":{"explanation":"The character '-' can function as a hyphen, an en dash, an em dash, or a minus sign, depending on context. A hyphen joins words or parts of words. An en dash indicates a range (e.g., pages 10-20). An em dash indicates a break in thought or separates clauses. As a minus sign, it indicates subtraction.","priority":"★★★"},"common_mistakes":{"examples":[{"incorrect":"Using a hyphen when an em dash is needed.","correct":"Using an em dash to set off a phrase.","note":"Make sure to use the correct length of dash for the intended purpose."}],"priority":"★★☆"},"related_words":{"derivatives":[],"phrasal_verbs":[],"priority":"★★★"},"level_frequency":{"CEFR":"N/A","frequency_google_ngram":"Very high frequency","priority":"★☆☆"},"readability_explanation":{"level":"Beginner","text":"The hyphen (-) is a punctuation mark. It connects words together. For example, 'well-behaved' uses a hyphen. The dash (— or –) is longer and shows a break in a sentence.","priority":"★★★"},"example_sentences":[{"sentence":"Use a hyphen in 'state-of-the-art'.","translation":"「最新技術の」にハイフンを使用します。","type":"example","meaning_category":"hyphen"},{"sentence":"The en dash (–) is used to indicate a range, e.g., 2010–2020.","translation":"中ダッシュ (–) は、範囲を示すために使用されます。例: 2010–2020。","type":"example","meaning_category":"en dash"},{"sentence":"The em dash (—) can set off a phrase — like this.","translation":"長ダッシュ (—) は、フレーズを区切ることができます — このように。","type":"example","meaning_category":"em dash"},{"sentence":"5 - 3 = 2","translation":"5 - 3 = 2","type":"example","meaning_category":"minus sign"}]}