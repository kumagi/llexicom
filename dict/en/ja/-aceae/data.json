{"word":"-aceae","priority":"☆☆☆","meanings":[{"part_of_speech":"suffix","definition":"植物の科名を作る接尾辞","english_definition":"A suffix used to form the names of botanical families.","examples":[{"sentence":"Rosaceae (バラ科)","translation":"Rosaceae (バラ科)"},{"sentence":"Fabaceae (マメ科)","translation":"Fabaceae (マメ科)"},{"sentence":"Poaceae (イネ科)","translation":"Poaceae (イネ科)"}],"collocations":[],"synonyms":[],"antonyms":[]}],"etymology":{"value":"ラテン語に由来する植物の科名を作るための標準的な接尾辞。","priority":"★★★"},"pronunciation":{"ipa":"/ˌeɪˈsiːiː/","syllables":"a-ce-ae"},"inflection":{},"usage_notes":{"explanation":"植物学において、科の名称は、属名の語幹にこの接尾辞を加えることによって形成されます。","priority":"★★★"},"common_mistakes":{},"related_words":{"derivatives":[],"phrasal_verbs":[]},"level_frequency":{},"readability_explanation":{"level":"C2","text":"This suffix is specific to botanical nomenclature and is not commonly used outside of scientific contexts. Understanding this suffix is essential for anyone studying botany or related fields."},"example_sentences":[{"sentence":"The family Solanaceae includes plants such as tomatoes and potatoes.","translation":"ナス科にはトマトやジャガイモなどの植物が含まれます。","type":"example","meaning_category":"植物の科の例"}]}