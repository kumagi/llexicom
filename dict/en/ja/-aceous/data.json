{"word":"-aceous","priority":"★☆☆","meanings":[{"part_of_speech":"suffix","definition":"～のような、～に属する、～から成る (形容詞を作る接尾辞。生物学、特に植物学でよく用いられる)","english_definition":"of or pertaining to; resembling; of the nature of; composed of (a suffix forming adjectives, especially in biology, particularly botany).","examples":[{"sentence":"herbaceous (草本のような、草質の)","translation":"herbaceous（草本のような）"},{"sentence":"crustaceous (甲殻類の、甲殻から成る)","translation":"crustaceous（甲殻類の）"},{"sentence":"arenaceous (砂質の、砂のような)","translation":"arenaceous（砂質の）"}],"collocations":["herbaceous plant (草本植物)","crustaceous animals (甲殻類動物)","arenaceous soil (砂質土壌)"],"synonyms":["-like","-ish","-al","-ar","-ine"],"antonyms":[]}],"etymology":{"value":"ラテン語の「-aceus」に由来し、「～から成る、～に属する」という意味を持つ形容詞を作る接尾辞。これは「-a-」(連結母音) + 「-ceus」(「～から作られた、～に似た」)から構成される。","priority":"★★☆"},"pronunciation":{"ipa":"/ˈeɪʃəs/","syllables":"a-ce-ous (3 syllables)"},"inflection":{"notes":"形容詞を作る接尾辞なので、活用形はありません。"},"usage_notes":{"explanation":"「**-aceous**」は、形容詞を作る接尾辞で、主に生物学、特に植物学の分野で用いられます。ある物質や特徴に「似ている」「属する」「から成る」といった意味合いを付加します。例えば、「herbaceous」は「草本のような」という意味になり、「herb」（草）に似た性質を持つことを示します。専門用語でよく見られる接尾辞ですが、日常会話ではあまり使われません。","priority":"★★☆"},"common_mistakes":{"examples":[{"incorrect":"✗ He described the rock as rockaceous.","correct":"✓ He described the rock as arenaceous.","note":"「rockaceous」という単語は一般的ではありません。砂質の岩を指す場合は「arenaceous」がより適切な表現です。"},{"incorrect":"✗ The plant was very herb.","correct":"✓ The plant was very herbaceous.","note":"「herb」は名詞（ハーブ、薬草）であり、形容詞として使う場合は「herbaceous」（草本のような）という形にする必要があります。"}],"priority":"★★☆"},"related_words":{"derivatives":["herbaceousness (草本性)","crustaceousness (甲殻性)","arenaceousness (砂質性)"],"idioms_phrases":[],"priority":"★☆☆"},"level_frequency":{"CEFR":"C2 (非常に専門的な語彙)","frequency_google_ngram":"低頻度 (Low frequency) - 専門用語で使われることが多い。","priority":"★☆☆"},"readability_explanation":{"level":"C1","text":"「**-aceous**」は、単語の最後に付いて、その単語が「～のような」「～に属する」という意味を持つことを示す接尾辞です。主に科学的な分野、特に生物学や植物学で使われます。例えば、「herbaceous」は「草のような」という意味です。日常会話ではあまり使いませんが、専門的な文章を読むときには知っておくと役立ちます。"},"example_sentences":[{"sentence":"The soil was **arenaceous**, making it difficult to grow certain plants.","translation":"その土壌は砂質で、特定の植物を育てるのが難しかった。","type":"adjective","meaning_category":"砂質の"},{"sentence":"**Herbaceous** plants die back to the ground each year.","translation":"草本植物は毎年地上部が枯れる。","type":"adjective","meaning_category":"草本のような"},{"sentence":"Lobsters are **crustaceous** animals with hard exoskeletons.","translation":"ロブスターは硬い外骨格を持つ甲殻類動物である。","type":"adjective","meaning_category":"甲殻類の"}]}