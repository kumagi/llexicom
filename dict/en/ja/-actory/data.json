{"word":"-actory","priority":"N/A","meanings":[{"part_of_speech":"suffix","definition":"～場、～所、～工場 (場所や施設を示す接尾辞)","english_definition":"a place or establishment for doing something (suffix indicating a place or facility)","examples":[{"sentence":"factory (工場)","translation":"工場"},{"sentence":"olfactory (嗅覚の)","translation":"嗅覚の (嗅覚器のある場所)"},{"sentence":"satisfactory (満足のいく)","translation":"満足のいく (十分な場所)"}],"collocations":[],"synonyms":["-orium","-ary","-ery"],"antonyms":[]}],"etymology":{"value":"ラテン語の「-faciō」（作る、行う）に由来する。場所や施設を示す接尾辞として、英語に取り入れられた。","priority":"★★☆"},"pronunciation":{"ipa":"/ˌækt(ə)ri/","syllables":"ac-to-ry"},"inflection":{"notes":"接尾辞なので活用はありません。"},"usage_notes":{"explanation":"「**-actory**」は、名詞の語尾に付いて、特定の活動が行われる場所や施設を示す接尾辞です。ただし、この接尾辞を持つ単語はそれほど多くありません。より一般的な接尾辞としては、「-ory」、「-ary」、「-ery」などがあります。","priority":"★★☆"},"common_mistakes":{"examples":[{"incorrect":"✗ creatactory","correct":"✓ factory","note":"「creatactory」のような、存在しない単語を作らないようにしましょう。接尾辞「-actory」が使われる単語は限られています。"}],"priority":"★☆☆"},"related_words":{"derivatives":["factory (工場)","manufactory (製造所)","olfactory (嗅覚の)","satisfactory (満足のいく)"],"priority":"★★★"},"level_frequency":{"CEFR":"N/A","frequency_google_ngram":"低頻度 (Low frequency) - 単独の単語としては存在せず、接尾辞として使われるため。","priority":"★☆☆"},"readability_explanation":{"level":"C1","text":"「**-actory**」は、単独の単語ではなく、他の語に付いて新しい単語を作る接尾辞です。この接尾辞は、ある活動が行われる場所や施設を示す意味を持ちます。例えば、「factory（工場）」は物を製造する場所、「olfactory（嗅覚の）」は嗅覚に関わる場所や器官を指します。ただし、この接尾辞を持つ単語は多くありません。英語学習においては、単語の構成要素として理解しておくと、未知の単語の意味を推測するのに役立つことがあります。"},"example_sentences":[{"sentence":"A **factory** is a building where goods are manufactured.","translation":"工場は製品が製造される建物です。","type":"example","meaning_category":"場所"},{"sentence":"The **olfactory** nerve is responsible for the sense of smell.","translation":"嗅神経は嗅覚を司っています。","type":"example","meaning_category":"場所・器官"},{"sentence":"The results were **satisfactory**.","translation":"結果は満足のいくものでした。","type":"example","meaning_category":"程度"}]}