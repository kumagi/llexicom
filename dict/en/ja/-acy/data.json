{"word":"-acy","priority":"★★☆","meanings":[{"part_of_speech":"suffix","definition":"～の状態、～の性質、～の地位、～の団体、～の制度","english_definition":"a suffix denoting state, condition, quality, office, body, or system.","examples":[{"sentence":"democracy (民主主義)","translation":"民主主義"},{"sentence":"accuracy (正確さ)","translation":"正確さ"},{"sentence":"privacy (プライバシー)","translation":"プライバシー"},{"sentence":"celibacy (独身主義)","translation":"独身主義"},{"sentence":"diplomacy (外交)","translation":"外交"},{"sentence":"piracy (海賊行為)","translation":"海賊行為"}],"collocations":["democracy (民主主義)","bureaucracy (官僚主義)","accuracy (正確さ)","privacy (プライバシー)","legacy (遺産)","supremacy (至上)","celibacy (独身)","fallacy (誤謬)","intimacy (親密さ)","piracy (海賊行為)"],"synonyms":[],"antonyms":[]}],"etymology":{"value":"古フランス語の「-acie」または直接ラテン語の「-acia」から来ており、ギリシャ語の「-akia」に由来する。これは名詞を作る接尾辞で、状態、質、地位、または集団を表す。","priority":"★★☆"},"pronunciation":{"ipa":"/əsi/","syllables":"a-cy"},"inflection":{},"usage_notes":{"explanation":"「-acy」は、名詞を形成するために使用される接尾辞です。通常、抽象名詞を作り、ある状態、性質、地位、団体、または制度を表します。例えば、「democracy」は「demos」（人々）と「-cracy」（支配）を組み合わせたもので、「人々による支配」を意味します。同様に、「accuracy」は「accurate」（正確な）から派生し、「正確さ」という性質を表します。この接尾辞は、多くの英語の単語に見られ、その単語の意味を理解する上で役立ちます。","priority":"★★★"},"common_mistakes":{"examples":[{"incorrect":"✗ accuraty","correct":"✓ accuracy","note":"形容詞 accurate (正確な) と名詞 accuracy (正確さ) を混同しないように。"},{"incorrect":"✗ privatecy","correct":"✓ privacy","note":"形容詞 private (私的な) と名詞 privacy (プライバシー) を混同しないように。"}],"priority":"★★☆"},"related_words":{"derivatives":["-cracy (～政治、～主義)","-ocracy (～政治、～主義)","-acity (～の性質、～の傾向)"],"phrasal_verbs":[],"priority":"★★☆"},"level_frequency":{"CEFR":"B2-C1 (単語によって異なる)","frequency_google_ngram":"中頻度 (Medium frequency) - 接尾辞を含む単語の頻度は様々。","priority":"★☆☆"},"readability_explanation":{"level":"C1","text":"「-acy」は、単語の最後に付いて、その単語が「状態」や「性質」、「地位」や「制度」を表す名詞であることを示す接尾辞です。例えば、「democracy」は「民主主義」という政治体制を、「accuracy」は「正確さ」という性質を表します。この接尾辞が付いた単語は、少し難しい単語が多いですが、意味を理解する手がかりになります。"},"example_sentences":[{"sentence":"The country transitioned to a **democracy** after years of authoritarian rule.","translation":"その国は長年の権威主義的支配の後、民主主義に移行した。","type":"noun","meaning_category":"制度"},{"sentence":"The **accuracy** of the data is crucial for making informed decisions.","translation":"データの正確さは、情報に基づいた意思決定を行うために非常に重要である。","type":"noun","meaning_category":"性質"},{"sentence":"The **legacy** of the ancient civilization is still visible today.","translation":"古代文明の遺産は今日でも見ることができる。","type":"noun","meaning_category":"状態"},{"sentence":"The company values the **privacy** of its customers.","translation":"その会社は顧客のプライバシーを重視している。","type":"noun","meaning_category":"状態"},{"sentence":"His **diplomacy** skills were essential in resolving the conflict.","translation":"彼の外交手腕は紛争解決に不可欠だった。","type":"noun","meaning_category":"団体・制度"}]}