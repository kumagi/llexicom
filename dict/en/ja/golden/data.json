{"word":"golden","priority":"★★☆","meanings":[{"part_of_speech":"adjective","definition":"黄金の、金色の","english_definition":"made of or resembling gold.","examples":[{"sentence":"She wore a golden necklace.","translation":"彼女は金のネックレスをしていた。"},{"sentence":"The sunset painted the sky with golden hues.","translation":"夕焼けが空を金色に染めた。"}],"collocations":["golden hair (金髪)","golden jewelry (金の装飾品)","golden sunset (黄金の夕焼け)"],"synonyms":["gold","gilded","auric"],"antonyms":["silver","leaden"]},{"part_of_speech":"adjective","definition":"貴重な、価値のある、素晴らしい","english_definition":"very valuable or precious.","examples":[{"sentence":"This is a golden opportunity.","translation":"これは絶好の機会だ。"},{"sentence":"He gave me some golden advice.","translation":"彼は私に貴重な助言をくれた。"}],"collocations":["golden opportunity (絶好の機会)","golden age (黄金時代)","golden rule (黄金律)","golden memories (かけがえのない思い出)"],"synonyms":["precious","valuable","excellent","splendid","superb"],"antonyms":["worthless","inferior"]},{"part_of_speech":"adjective","definition":"幸せな、繁栄した","english_definition":"happy and prosperous.","examples":[{"sentence":"They lived in a golden era of peace and prosperity.","translation":"彼らは平和と繁栄の黄金時代に生きていた。"}],"collocations":["golden years (人生の黄金期、退職後の年月)"],"synonyms":["happy","prosperous","flourishing"],"antonyms":["unhappy","unsuccessful"]},{"part_of_speech":"adjective","definition":"(記念日などが)金婚式の","english_definition":"relating to a fiftieth anniversary, especially a wedding anniversary.","examples":[{"sentence":"They celebrated their golden wedding anniversary.","translation":"彼らは金婚式を祝った。"}],"collocations":["golden wedding (金婚式)"],"synonyms":[],"antonyms":[]}],"etymology":{"value":"古英語の「gylden」に由来し、「金でできている」または「金色に似ている」という意味を持つ。これは「gold」（金）という単語から派生した。比喩的な意味での「貴重な」や「素晴らしい」という意味は、金の価値の高さから来ている。","priority":"★★☆"},"pronunciation":{"ipa":"/ˈɡoʊldən/","syllables":"gol-den"},"inflection":{"notes":"形容詞なので、比較級・最上級の変化はありません。"},"usage_notes":{"explanation":"「**golden**」は、文字通りには「金でできている」または「金色」を意味しますが、比喩的には「非常に価値のある」「素晴らしい」「幸せな」といった意味合いで使われます。特に「golden opportunity」（絶好の機会）や「golden age」（黄金時代）といった表現はよく用いられます。また、結婚50周年を祝う「金婚式」を意味する「golden wedding」という表現もあります。","priority":"★★★"},"common_mistakes":{"examples":[{"incorrect":"✗ She has gold hair.","correct":"✓ She has golden hair.","note":"髪の色を表現する場合は、通常「golden hair」と言います。「gold hair」も間違いではありませんが、「golden hair」の方が一般的です。"},{"incorrect":"✗ This is a gold chance.","correct":"✓ This is a golden opportunity.","note":"「絶好の機会」を意味する場合は、「golden opportunity」という決まった言い方をします。「gold chance」は一般的ではありません。"}],"priority":"★★☆"},"related_words":{"derivatives":["gild (動詞: 金メッキする)","gold (名詞: 金)"],"idioms_phrases":["golden handshake (退職金、解雇手当)","golden parachute (経営者が解任された場合に支払われる高額な退職金)","golden rule (黄金律)"],"priority":"★★☆"},"level_frequency":{"CEFR":"B1","frequency_google_ngram":"比較的高頻度 (Relatively high frequency)","priority":"★☆☆"},"readability_explanation":{"level":"B1","text":"「**golden**」は、文字通りには「金色」という意味ですが、それ以外にも「とても価値がある」や「素晴らしい」という意味で使われることが多い単語です。例えば、「golden opportunity」は「めったにない良い機会」という意味になります。また、「golden age」は「一番良い時代」という意味で使われます。結婚50周年を「golden wedding」と言うことも覚えておきましょう。"},"example_sentences":[{"sentence":"The **golden** leaves fell from the trees.","translation":"木の葉が金色に色づいて落ちてきた。","type":"adjective","meaning_category":"黄金の・金色の"},{"sentence":"He seized the **golden** opportunity to travel the world.","translation":"彼は世界を旅するという絶好の機会をつかんだ。","type":"adjective","meaning_category":"貴重な・価値のある"},{"sentence":"They are enjoying their **golden** years in retirement.","translation":"彼らは退職後の黄金期を楽しんでいる。","type":"adjective","meaning_category":"幸せな・繁栄した"},{"sentence":"They celebrated their **golden** wedding anniversary with family and friends.","translation":"彼らは家族や友人と金婚式を祝った。","type":"adjective","meaning_category":"金婚式の"}]}